src/sources/dmm.cpp

```diff
@@ -2027,3 +2030,147 @@ void DMM::readVC870Continuous( const QByteArray & data, int /*id*/, ReadEvent::D
         m_error = tr( "Parser errors on %1" ).arg(m_device);
     } // else
 }
+
+void DMM::readDO32122Continuous( const QByteArray & data, int /*id*/, ReadEvent::DataFormat /*df*/ )
```

Comment:
Please use the Qt function Q_UNUSED instant of /*id*/ and so on.
https://doc.qt.io/qt-6/qtglobal.html#Q_UNUSED